pages/windows/msedge.md

```diff
@@ -3,7 +3,7 @@
 > Modern web browser developed by Microsoft based on the Chromium web browser developed by Google.
 > This command is available instead as `microsoft-edge` for other platforms.
 > Note: Additional command arguments from `chromium` may also be usable to control Microsoft Edge.
-> More information: <https://microsoft.com/edge>.
+> More information: <https://www.microsoft.com/edge/>.
```

Comment:
Any chance you could find a link that documents the command line usage? The front page of the site is not useful information with regards to command line usage.

Reply:
Alright, will check.

Reply:
@Managor do you think this is good? https://peter.sh/experiments/chromium-command-line-switches/

Reply:
@Managor meh but not the best, https://learn.microsoft.com/en-us/deployedge/

Reply:
Very meh. We'll just leave it on the front page since most of the browser alias pages (see `tldr opera` and `tldr vivaldi` as examples) just point to the front pages of the website

Reply:
> Very meh. We'll just leave it on the front page since most of the browser alias pages (see `tldr opera` and `tldr vivaldi` as examples) just point to the front pages of the website

okay, so we close the PR or?